Default Programmable Buffer Length in rk3399's gmac is not suitable
for MTUs higher than 1498. The easiest solution would be to disable
offloading but it is not ideal as it disables hardware checksumming.

This patch sets pbl to 0x4 which is good for the most popular MTU
value of 1500.

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index 0125fc6e..e8b26881 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -286,6 +286,7 @@
 		resets = <&cru SRST_A_GMAC>;
 		reset-names = "stmmaceth";
 		rockchip,grf = <&grf>;
+		snps,pbl = <0x4>;
 		status = "disabled";
 	};

